www/package.json

```diff
@@ -110,6 +111,24 @@
   "license": "MIT",
   "main": "n/a",
   "private": true,
+  "babel": {
+    "presets": ["babel-preset-gatsby"]
```

Comment:
@wardpeet, @pvdz told me that you might know what's up. For some reason, [Lingui](https://lingui.js.org/tutorials/setup-react.html) requires this configuration in order for the CLI to run properly and let us do `lingui extract`. But it shouldn't be necessary, since it's the default included in gatsby.

Reply:
(This was in relation to webpack, but it turns out this was actually needed because lingui couldn't figure out our preset without a hint like this)